Viper Practice/src/controllers/Cells/TaskTableViewCell.swift

```diff
@@ -26,8 +26,8 @@ class TaskTableViewCell: UITableViewCell {
         // Configure the view for the selected state
     }
     
-    func configureCell(task: Task){
```

Comment:
Class name is still TaskTableViewCell, should it be renamed as well?

Reply:
all classes names renamed from Task to Fruit